### PR TITLE
RIA-4360 OOT Decisions: amends to Summary page for appeal cannot continue decision

### PR DESCRIPTION
**Pull request by gavinslilley**
### JIRA link (if applicable) ###
[RIA-4360](https://tools.hmcts.net/jira/browse/RIA-4360)

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: app/utils/application-state-utils.ts [hunk 1]
```diff
@@ -268,7 +280,17 @@ interface DoThisNextSection {
  * @param req the request containing the session and appeal status
  */
 function getAppealStatus(req: Request) {
-  return (req.session.appeal.application.isAppealLate && req.session.appeal.appealStatus === 'appealSubmitted') ? 'lateAppealSubmitted' : req.session.appeal.appealStatus;
+  if (req.session.appeal.application.isAppealLate) {
+    if (req.session.appeal.appealStatus === 'appealSubmitted') {
+      return 'lateAppealSubmitted';
+    }
+    if (req.session.appeal.outOfTimeDecisionType) {
+      return 'lateAppealEnded';
```
Comment (t0nin0s) [inline app/utils/application-state-utils.ts:288]: to be more precise shouldn't it be 'lateAppealRejected'?

File: test/e2e-test/pages/overview-page/overview-page.ts [hunk 2]
```diff
@@ -50,6 +50,13 @@ module.exports = {
       I.seeInSource(`${i18n.pages.overviewPage.doThisNext.clarifyingQuestions.respondByTextAskForMoreTime}`);
     });
 
+    Then(/^I should see the 'do this next section' for 'Your appeal cannot continue'$/, () => {
```
Comment (t0nin0s) [inline test/e2e-test/pages/overview-page/overview-page.ts:53]: If we can let's avoid adding more steps and keep it as generic as possible, I think there's a step you could use to check subheadings

File: app/utils/application-state-utils.ts (hunk 1 above)
Comment (gavinslilley) [reply app/utils/application-state-utils.ts:288]: Agree, updated

File: test/e2e-test/pages/overview-page/overview-page.ts (hunk 2 above)
Comment (gavinslilley) [reply test/e2e-test/pages/overview-page/overview-page.ts]: agreed, removed

**Review by jkumar-moj — APPROVED** [review]

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Pull request merged**